docs/design.html

```diff
@@ -385,8 +385,10 @@ <h3 class="anchor-heading"><a id="replication" class="anchor-link"></a><a href="
     have the option of either waiting for the message to be committed or not, depending on their preference for tradeoff between latency and durability. This preference is controlled by the acks setting that the
```

Comment:
suggestion while in the vicinity
```suggestion
    have the option of either waiting for the message to be committed or not, depending on their preference for tradeoff between latency and durability. This preference is controlled by the <code>acks</code> setting that the
```